glib/src/param_spec.rs

```diff
@@ -475,12 +480,12 @@ macro_rules! define_builder {
             $($field_id: Option<$field_ty>),*
         }
         impl<'a> $builder_type<'a> {
-            /// Default: `self.name`
+            /// Default: `None`
```

Comment:
Technically the default is still `self.name` as that's what will be used as default from C

Reply:
It is actually a bit more subtle than that, I have updated the docs. Let me know if you think this is better.